### DISCUSSION TITLE
Screen on Kobra Neo not responding while and after installing firmware

### DISCUSSION THREAD

**Discussion by DaxMein** [1coderookie/Klipper4KobraGoNeo#11] [General]
Hello

I am a total noob when it comes to all that stuff and I basically created my Github account to make this post.
I do not get the  "Updating firmware. Please wait." screen and after waiting 5-10 minutes turning of the printer, remove the card and turn it on again, it stay black.

any suggestions?

**Comment by jokubasver** [comment]
> Hello
> 
> I am a total noob when it comes to all that stuff and I basically created my Github account to make this post.
> I do not get the  "Updating firmware. Please wait." screen and after waiting 5-10 minutes turning of the printer, remove the card and turn it on again, it stay black.
> 
> any suggestions?

That's normal, Klipper doesn't support the LCD used in the Kobra Neo. If the display stays blank - you're more than likely running Klipper just fine.

**Reply by DaxMein** [reply]
yes actually now it works, but I wonder why its mentioned, that it will show "updating firmware"
is there any easy way to get any screen running?

**Reply by jokubasver** [reply]
Not sure why it says that in there docs, it never did for me.

Screen options for Klipper include just using the web interface (Mainsail, Fluidd, Octoprint). Another option is Klipperscreen along with a HDMI screen connected to your Raspberry Pi/other SBC, or a smartphone/tablet.

**Reply by 1coderookie** [reply]
It showed that message at the first Neo I flashed back in the days, nowadays it doesn't anymore though, the screen is just black the whole time.  Dunno why it even appeard back then, but it did.. So I'll change that at the page, I forgot to do so tbh..
